File: CPP/Tim.cpp
//NamaTim, JumlahTim

#pragma once
#include <iostream>
#include <string>

using namespace std;

class Tim{
    private:
        string NamaTim;
        int JumlahTim;
    
    public:
        Tim(){
            NamaTim = "";
            JumlahTim = 0;
        };
        Tim(string nama, int jumlah){
            NamaTim = nama;
            JumlahTim = jumlah;
        }

        //setter
        void setNamaTim(string nama){
            NamaTim = nama;
        }
        void setJumlahTim(int jumlah){
            JumlahTim = jumlah;
        }

        //getter
        string getNamaTim(){
            return NamaTim;
        }
        int getJumlahTim(){
            return JumlahTim;
        }
        ~Tim(){
        }
};

File: JAVA/Tim.java
//NamaTim, JumlahTim

public class Tim{
    private String NamaTim;
    private int JumlahTim;

    public Tim(){
        this.NamaTim = "";
        this.JumlahTim = 0;
    }

    public Tim(String nama, int jumlah){
        this.NamaTim = nama;
        this.JumlahTim = jumlah;
    }

    //setter
    public void setNamaTim(String nama){
        this.NamaTim = nama;
    }

    public void setJumlahTim(int jumlah){
        this.JumlahTim = jumlah;
    }

    //getter
    public String getNamaTim(){
        return this.NamaTim;
    }

    public int getJumlahTim(){
        return this.JumlahTim;
    }
}